src/ucp/core/ucp_mm.c

```diff
@@ -175,14 +175,21 @@ static ucs_status_t ucp_mem_alloc(ucp_context_h context, size_t length,
 }
 
 
-ucs_status_t ucp_mem_map(ucp_context_h context, void **address_p, size_t length,
-                         unsigned flags, ucp_mem_h *memh_p)
+ucs_status_t ucp_mem_map(ucp_context_h context, ucp_mem_map_params_t *params,
```

Comment:
I removed the const from the signature since the ucp_mem_map function may change the address field inside the params arg.